util/src/cryptopp_digest_creator.cpp

```diff
@@ -58,7 +57,7 @@ bool CryptoppDigestCreator::compute(const char* input,
   return true;
 }
 
-void CryptoppDigestCreator::update(const char* data, size_t len) {
+void CryptoppDigestCreator::updateDigest(const char* data, const size_t len) {
```

Comment:
remove the const here. Why did u added this?

Reply:
`const` is added with an intention of "do not modify the input parameters".

Reply:
I think const is not needed here.

Reply:
Removed.